### PR TITLE
talk about let helper in references example

### PR DESCRIPTION
**Pull request by cherifGsoul**
This is for #488

### CODE REVIEW

File: docs/two-way.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ Click on one of the list items below and watch as its text appears in the input
 @demo demos/can-stache-bindings/two-way.html
 @codepen
 
-This demo can be expressed a bit easier with the references scope:
+This demo can be expressed a bit easier with the references scope by using [can-stache.helpers.let let] helper:
```
Comment (chasenlehara) [inline docs/two-way.md:90]: I’d write “using the let helper”

Does the demo need to be updated too, or was that already done?

Comment (cherifGsoul) [reply docs/two-way.md:90]: for this ```I’d write “using the let helper” ``` still keep the link to the helper doc page?
For the demo, I don't see any demo for let helper in the docs!

Comment (cherifGsoul) [reply docs/two-way.md:90]: @chasenlehara The page of this docs has been changed, I wonder if we still need to fix this issue? https://canjs.com/doc/can-stache-bindings.twoWay.html

Comment (chasenlehara) [reply docs/two-way.md:90]: I would double check with @justinbmeyer since he made changes to the docs. It looks like this section has been removed.

**Pull request closed without merging**